zuul.d/jobs.yaml

```diff
@@ -760,13 +760,42 @@
       ansible_test_command: network-integration
       ansible_test_integration_targets: "openvswitch_.*"
 
+- job:
+    name: ansible-test-network-unit-openvswitch
+    abstract: true
+    dependencies:
+      - name: build-ansible-collection
+        soft: true
+    parent: ansible-network-openvswitch-appliance
+    pre-run:
+      - playbooks/ansible-test-base/pre.yaml
+      - playbooks/ansible-test-network-integration-base/pre.yaml
+    run: playbooks/ansible-test-base/run.yaml
+    post-run:
+      - playbooks/ansible-test-network-integration-base/post.yaml
+      - playbooks/ansible-test-base/post.yaml
+    required-projects:
+      - name: github.com/ansible/ansible
+    timeout: 3600
+    vars:
+      ansible_test_command: network-units
+      ansible_test_integration_targets: "test_openvswitch_.*"
+
+
 - job:
     name: ansible-test-network-integration-openvswitch-python27
     parent: ansible-test-network-integration-openvswitch
     nodeset: openvswitch-2.9.0-python27
     vars:
       ansible_test_python: 2.7
 
+- job:
+    name: ansible-test-network-unit-openvswitch-python27
+    parent: ansible-test-network-unit-openvswitch
+    nodeset: openvswitch-2.9.0-python27
```

Comment:
so this nodeset is going to be incorrect, as it will also boot the appliance, which isn't needed for unit tests.

What we'll need to do is create something like:

```
- nodeset:
    name: controller-python27
    nodes:
      - name: ubuntu-bionic
        label: ubuntu-bionic-1vcpu
    groups:
      - name: controller
        nodes:
          - ubuntu-bionic
```
then reference it here. You can add it into zuul.d/nodesets.yaml